oidc-signature-extension.md

```diff
@@ -1,10 +1,14 @@
+![Logo](docs/images/oidc-logo.png)
+
 # Signature Extension for OpenID Connect
 
-### Version: 1.0 - draft 01 - 2021-05-21
+### Version: 1.0 - draft 02 - 2023-04-05
 
 ## Abstract
 
-This specification defines an extension to OpenID Connect to facilitate the use cases where a Relying Party sends a "Signature Request" to an OpenID Provider. A signature request is an extension of an OpenID Connect authentication request where a SignatureRequest object is passed as a request parameter or a Request Object.
+This specification defines an extension to OpenID Connect to facilitate the use cases where a Relying Party sends a
```

Comment:
Nitpick: I think the wording should be "... facilitate the use **case** where ...." or "... facilitate use cases where ...". 

Something feels slightly off with "facilitate **the** use cases where ... ".